OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -530,6 +531,10 @@ public boolean isWhatsNewEnabled() {
         return getBoolean(WHATS_NEW_ENABLED, false);
     }
 
+    public boolean isMyVideosEnabled() {
+        return getBoolean(MY_VIDEOS_ENABLED, false);
```

Comment:
As per discussion with Saeed, it should be off by-default as its not a necessary feature.